server/repository/src/db_diesel/program_requisition/program_supplier.rs

```diff
@@ -9,12 +9,20 @@ use crate::{
         program_row::program::dsl as program_dsl, store_row::store::dsl as store_dsl,
     },
     diesel_macros::apply_equal_filter,
+    name_oms_fields, name_oms_fields_alias,
     repository_error::RepositoryError,
-    EqualFilter, Name, NameFilter, NameLinkRow, NameRepository, NameRow, NameStoreJoinRow,
-    ProgramRow, StorageConnection, StoreRow,
+    EqualFilter, Name, NameFilter, NameLinkRow, NameOmsFieldsRow, NameRepository, NameRow,
+    NameStoreJoinRow, ProgramRow, StorageConnection, StoreRow,
 };
 
-pub type ProgramSupplierJoin = (NameRow, NameLinkRow, NameStoreJoinRow, StoreRow, ProgramRow);
+pub type ProgramSupplierJoin = (
```

Comment:
Program Supplier query uses name query as a base, so also added the name_oms_fields in here